public/gatk-utils/src/main/java/org/broadinstitute/gatk/utils/MannWhitneyU.java

```diff
@@ -208,9 +217,16 @@ public void add(Number n, USet set) {
      */
     @Requires({"m > 0","n > 0"})
     @Ensures({"! Double.isNaN(result)", "! Double.isInfinite(result)"})
-    private static double getZApprox(int n, int m, long u) {
+    private static double getZApprox(int n, int m, double u, Map<Integer, Integer> tieStructure) {
         double mean = ( ((long)m)*n+1.0)/2;
```

Comment:
Is there some off-by-one issue here?  The mean value of the normal distribution should be nm/2, not (nm + 1) / 2.